<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor ACOES: Gestor de apadrinamientos</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tBecados">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>731</width>
      <height>351</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Estado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha Nacimiento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha de Alta</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Grado</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Colonia Nacimiento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Colonia Residencia</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Colegio</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>Constantino</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>de la Cruz García Dominguez</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>Honduras</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>29/02/2012</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>04/03/2012</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>ioqc</string>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="7">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="8">
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="bAtras">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Atras</string>
    </property>
   </widget>
   <widget class="QLabel" name="lBusqueda">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Busqueda:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bActualizar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bAnadirBecado">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Añadir Becado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bEditarBecado">
    <property name="geometry">
     <rect>
      <x>649</x>
      <y>440</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar Becado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bProyectos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Proyectos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lNombre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="lApellidos">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellidos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eNombre">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eApellidos">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lEstado">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCNacimiento">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>C.  Nacimiento</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cEstado">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cCNacimiento">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lColegio">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Colegio</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cCResidencia">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cColegio">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lCResidencia">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C. Residencia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bApadrinamientos">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista de Apadrinamientos</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
